index.js

```diff
@@ -0,0 +1,8 @@
+const bookmarks = document.querySelectorAll(
+  ".question-card .material-symbols-rounded"
```

Comment:
You can add the same data-js attribute to different elements, so you could select all bookmarks by one data-js attribute instead of a class.